lib/KnormRelations.js

```diff
@@ -321,10 +321,10 @@ class KnormRelations {
           row[this.formatFieldAlias(this.options.unique, { quote: false })];
 
         if (unique) {
-          if (this.parsedRows[unique]) {
-            parsedRow = this.parsedRows[unique];
+          if (this.parsedRows.has(unique)) {
```

Comment:
we can save an extra function call here if we first get the row and store it in a variable i.e. 
```js 
const row = this.parsedRows.get(unique); 
if (row) { parsedRow = row; }
```